src/openrct2/scripting/ScriptEngine.h

```diff
@@ -47,7 +47,7 @@ namespace OpenRCT2
 
 namespace OpenRCT2::Scripting
 {
-    static constexpr int32_t OPENRCT2_PLUGIN_API_VERSION = 73;
+    static constexpr int32_t OPENRCT2_PLUGIN_API_VERSION = 74;
```

Comment:
API version number needs to be bumped again due to https://github.com/OpenRCT2/OpenRCT2/commit/2fd3eb47e753c4a8c47ddddffdab9e61299609f3#diff-c39c33ccf0667447861459fe529a9cae5b800eecac138edae5c2dd69d3192796
```suggestion
    static constexpr int32_t OPENRCT2_PLUGIN_API_VERSION = 75;
```

I'm excited for this PR!